lib/ae_mdw/db/async_store.ex

```diff
@@ -0,0 +1,89 @@
+defmodule AeMdw.Db.AsyncStore do
+  @moduledoc """
+  Implementation of Store protocol to be used by asynchronous tasks.
+
+  Its operations have immediate in-memory effect (not persisted) and
+  is backed by a cache with TTL of 1 day.
+  """
+
+  alias AeMdw.Database
+  alias AeMdw.EtsCache
+
+  @derive AeMdw.Db.Store
+  defstruct tid: nil
+
+  @table_id :single_async_store
+  @ttl_minutes 24 * 60
+
+  @typep key() :: Database.key()
+  @typep record() :: Database.record()
+  @typep table() :: Database.table()
+  @opaque t() :: %__MODULE__{tid: :single_async_store}
```

Comment:
Can we use the same constant we defined before here (`@table_id`)? to avoid repeating constants that are supposed to match with each other

Reply:
it's due the compilation phase (it processes types before).